src/codegen.cpp

```diff
@@ -9825,8 +9825,14 @@ Error create_c_object_cache(CodeGen *g, CacheHash **out_cache_hash, bool verbose
     }
     cache_buf(cache_hash, compiler_id);
     cache_int(cache_hash, g->err_color);
+    cache_list_of_str(cache_hash, g->framework_dirs.items, g->framework_dirs.length);
+    cache_bool(cache_hash, g->libcpp_link_lib != nullptr);
+    cache_buf(cache_hash, g->zig_lib_dir);
+    cache_buf(cache_hash, &g->llvm_triple_str);
```

Comment:
Not this one, it is generated based on the other target information which is already in the hash.